zircon.core/src/commonTest/kotlin/org/hexworks/zircon/CharacterTilesAsString.kt

```diff
@@ -0,0 +1,34 @@
+package org.hexworks.zircon
+
+import org.hexworks.zircon.api.data.CharacterTile
+import org.hexworks.zircon.api.data.Position
+import org.hexworks.zircon.api.data.Size
+import org.hexworks.zircon.api.data.Tile
+import org.hexworks.zircon.api.graphics.TileComposite
+
+/**
+ * Converts a collection of character tiles (usually found at [TileComposite.tiles]) into a string.
+ *
+ * Note that this is just complicated enough to warrant its own unit tests (in jvmTest).
```

Comment:
I think you can remove this line as it was only intended as a comment to the reviewer, right?

Reply:
Not really, but I'll remove it anyway.